src/core/exception.filter.ts

```diff
@@ -142,6 +146,12 @@ export class ExceptionFilter implements GqlExceptionFilter {
         message: 'Failed to retrieve data from CORD database',
       };
     }
+    if (exs.some((e) => e instanceof SessionExpiredError)) {
+      return {
+        codes: ['SessionExpired', 'Transient', 'Database', 'Server'],
+        message: 'The CORD database session has expired',
```

Comment:
```suggestion
        message: 'The query to the database has expired',
```
maybe this would be better?